app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
it is better to use a simpler check
```suggestion
        if not vstr.get("vaccine", None):
```